### PR TITLE
Rename "Quake2" -> "Distortion"

### PR DESCRIPTION
**Pull request by engineer124**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

This came from my testing of camera work from MM. Quake2 is all about distorting the view in various shapes and sizes (some of which end up resembling a quake, but others don't). Examples of this in use is during the transformation between forms, taking the warp pad from the start of the dungeon to the boss-room, using a non-magic spin attack (that one is very small and hard to notice), hot rooms (unused in MM but the option is still there, you can see OoT for examples), and uses in other places such as z_play (some of the distortions look like song of soaring/song of time distortions, but it's hard to say without decomped code). Another example from OoT is when you first enter water with iron boots, the screen gets distorted for a few seconds before settling. This was obviously dropped in MM as there are no longer iron boots and instead is zora, and one would not expect a Zora to be disoriented in water when first entering.

One part added to this was fixing the name `SurfaceType_IsConveyor` -> `SurfaceType_GetConveyorType` from BgCheck. The code in view didn't make sense before as it only returned the conveyor speed if the surface was not a conveyor? Looking into it more using OoT, specifically this block:

https://github.com/zeldaret/oot/blob/0b8edc21c03a719d8f7874bc20204612d6d1c107/src/overlays/actors/ovl_player_actor/z_player.c#L9455-L9465

A value of 0 seems to be checking for an underwater current, while a value of 1 seems to be checking for a conveyor surface (both eventually translate into windSpeed that moves Player).

I doubled checked the MM version of this Player function from the link above in Mips2C and saw the same behaviour.

### CODE REVIEW

File: tools/actorfixer.py [hunk 1]
```diff
@@ -344,7 +344,7 @@
     "func_800C9CEC": "SurfaceType_IsHookshotSurface",
     "func_800C9D14": "SurfaceType_IsIgnoredByEntities",
     "func_800C9D50": "SurfaceType_IsIgnoredByProjectiles",
-    "func_800C9D8C": "SurfaceType_IsConveyor",
+    "func_800C9D8C": "SurfaceType_GetConveyorType",
```
Comment (AngheloAlf) [inline tools/actorfixer.py:347]: Don't forget to add the new rename you are adding
```suggestion
    "func_800C9D8C": "SurfaceType_GetConveyorType",
    "SurfaceType_IsConveyor": "SurfaceType_GetConveyorType",
```

Comment (AngheloAlf) [reply tools/actorfixer.py:347]: Are the functions you renamed used a much? Maybe those renames should be added to actorfixer too

Comment (engineer124) [reply tools/actorfixer.py:347]: This function is only used two more times in the entire game. One in `Player_UpdateCommon` which I assume you haven't gotten to yet, and in `ovl_Effect_Ss_Bubble` which is untouched and unreserved. So this function can honestly probably go from actorFixer

Comment (AngheloAlf) [reply tools/actorfixer.py:347]: In that case let's remove it lol

**Review by AngheloAlf — APPROVED** [review]

**Review by jpburnett — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]
Looks better! Nice.

**Pull request merged**